scalikejdbc-core/src/test/scala/scalikejdbc/interpolation/SQLSyntaxSpec.scala

```diff
@@ -16,7 +22,7 @@ class SQLSyntaxSpec extends FlatSpec with Matchers {
   it should "have #append" in {
     val s = SQLSyntax.eq(sqls"id", 123).append(sqls"and name is not null")
     s.value should equal(" id = ? and name is not null")
-    s.parameters should equal(Seq(123))
+    s.parameters.map(parameterExtractor) should equal(Seq(123))
```

Comment:
hmm, this is not a bin-incompatibility but an API breaking change. I'm not sure this is safe enough yet.